src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1AssessmentDomainEventService.kt

```diff
@@ -185,7 +185,7 @@ class Cas1AssessmentDomainEventService(
                 name = staffDetails.probationArea.description,
               ),
               cru = Cru(
-                name = cruService.cruNameFromProbationAreaCode(staffDetails.probationArea.code),
+                name = userService.getUserForRequest().apArea?.name.toString(),
```

Comment:
Although we do it alot, i'm not a fan of using userService.getUserForRequest() outside of the controller layer. Instead of doing this, could we please pass the UserEntity in from the calling AssessmentService.

Furthermore, i've noticed that the StaffUserDetails being resolved in AssessmentService.acceptAssessment are only being used by this domain event function, so instead we should stop loading the StaffUserDetails in  AssessmentService.acceptAssessment and instead load them in this function instead (which makes this function more like the other domain event functions you've modified)